kuksa-client/README.md

```diff
@@ -1,46 +1,46 @@
-# Kuksa VISS Client
+# kuksa-client
```

Comment:
You mention a little about naming conventions in the PR description, e.g. when we need dash and when we need underscore. This is a third case - when referring to the components in documentation. I am thinking if we need to describe our naming strategy or a summary of them somewhere, like a table on top level README

```
Component    | Folder       | Python package name
----------------------------------------------------
Kuksa Client | kuksa-client | kuksa_client
```
I do not have a strong opinion if we always to refer to the kuksa client as "kuksa-client" or as "Kuksa Client", as long as we are consistent. After we have adapated kuksa-client we may want to synch other kuksa.val components as well, as naming today is quite inconsistent

Reply:
Agreed for "Kuksa Client". I'm using this as a component name now.